### DISCUSSION TITLE
Does PNL include commissions?

### DISCUSSION THREAD

**Discussion by 0xdaduck** [nautechsystems/nautilus_trader#1919] [Q&A] [answered]
In the statistics obtained from engine.trader.generate_positions_report(), is the commission included in the pnl? In other words, is the commission or handling fee subtracted from the final calculated pnl?

**Comment by cjdsellers** [comment, marked as answer]
Hi @ffssea

Yes, the PnL calculations do account for commissions. In addition, total commissions are calculated for individual positions and aggregated for each currency.

Let me know if you need further details or clarification!

**Comment by 0xdaduck** [comment]
thanks many,  @cjdsellers 

I've been using Nautilus for less than a week, and I want to express my gratitude to the community and maintainers for providing such detailed examples. I do have a quick question:

In my strategy, I need to frequently check my current position. Is the following statement correct? `self.portfolio.account(self.instrument_id.venue)`

best regards,
